fl4health/clients/apfl_client.py

```diff
@@ -26,8 +26,7 @@ def __init__(
 
         self.model: ApflModule
         self.learning_rate: float
-        self.optimizer: torch.optim.Optimizer
-        self.local_optimizer: torch.optim.Optimizer
+        self.optimizers: Dict[str, torch.optim.Optimizer]
```

Comment:
Update optimizers attribute to be dict instead of a single optimizer which was necessary for the addition of ensemble models.